### PR TITLE
codersnotes.VerySleepy version 0.91

### PR DESCRIPTION
**Pull request by mhwdvs**
Submission containing materials of a third party: [codersnotes](http://www.codersnotes.com/sleepy/)/[VerySleepy](https://github.com/VerySleepy/verysleepy)

---

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/62674)

### CODE REVIEW

File: manifests/c/codersnotes/VerySleepy/0.91.0.0/codersnotes.VerySleepy.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: codersnotes.VerySleepy
+PackageVersion: 0.91.0.0
+PackageLocale: en-US
+Publisher: codersnotes
```
Comment (ItzLevvie) [inline manifests/c/codersnotes/VerySleepy/0.91.0.0/codersnotes.VerySleepy.locale.en-US.yaml:7]:
```suggestion
Publisher: codersnotes.com
```
This must match Control Panel.

File: manifests/c/codersnotes/VerySleepy/0.91.0.0/codersnotes.VerySleepy.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: codersnotes.VerySleepy
+PackageVersion: 0.91.0.0
```
Comment (ItzLevvie) [inline manifests/c/codersnotes/VerySleepy/0.91.0.0/codersnotes.VerySleepy.installer.yaml:5]:
```suggestion
PackageVersion: 0.91
```

File: manifests/c/codersnotes/VerySleepy/0.91.0.0/codersnotes.VerySleepy.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: codersnotes.VerySleepy
+PackageVersion: 0.91.0.0
```
Comment (ItzLevvie) [inline manifests/c/codersnotes/VerySleepy/0.91.0.0/codersnotes.VerySleepy.yaml:5]:
```suggestion
PackageVersion: 0.91
```

File: manifests/c/codersnotes/VerySleepy/0.91.0.0/codersnotes.VerySleepy.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: codersnotes.VerySleepy
+PackageVersion: 0.91.0.0
+PackageLocale: en-US
+Publisher: codersnotes
+PackageName: Very Sleepy
+License: GNU General Public License v2.0
+ShortDescription: 'Very Sleepy, a sampling CPU profiler for Windows '
```
Comment (ItzLevvie) [inline manifests/c/codersnotes/VerySleepy/0.91.0.0/codersnotes.VerySleepy.locale.en-US.yaml:10]:
```suggestion
ShortDescription: Very Sleepy is a free C/C++ CPU profiler for Windows systems.
```

File: manifests/c/codersnotes/VerySleepy/0.91.0.0/codersnotes.VerySleepy.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: codersnotes.VerySleepy
+PackageVersion: 0.91.0.0
+PackageLocale: en-US
+Publisher: codersnotes
+PackageName: Very Sleepy
+License: GNU General Public License v2.0
```
Comment (ItzLevvie) [inline manifests/c/codersnotes/VerySleepy/0.91.0.0/codersnotes.VerySleepy.locale.en-US.yaml:9]:
```suggestion
License: GPL-2.0-only
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**